src/callbacks.rs

```diff
@@ -104,3 +105,92 @@ pub trait ParseCallbacks: fmt::Debug + UnwindSafe {
         vec![]
     }
 }
+
+/// The same trait as above, but clonable so that it can be included as a
+/// field in Builder when the builder-clone feature is enabled such that
+/// Builder implements Clone.
+#[cfg(feature = "builder-clone")]
+pub trait ParseCallbacks:
```

Comment:
It is really unfortunate to duplicate all the trait here. Can we avoid this by using a macro instead?